OpenXmlPowerTools/PegBase.cs

```diff
@@ -1643,71 +2116,107 @@ public bool Into(Matcher toMatch,out string into)
                 return false;
             }
         }
-        public bool Into(Matcher toMatch,out PegBegEnd begEnd)
+
+        public bool Into(Matcher toMatch, out PegBegEnd begEnd)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider refactoring this method in order to remove the need for this 'out' modifier.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)